docs/getting_started.md

```diff
@@ -1,238 +1,204 @@
+We recommend working through this guide completely to ensure everything is working in your environment. After the sample is working, follow the [bootstrap instructions](../bootstrap/README.md) to convert the ***diabetes_regression*** sample into a starting point for your project.
 
-## Decide best option to copy repository code
+- [Setting up Azure DevOps](#setting-up-azure-devops)
+- [Get the code](#get-the-code)
+- [Create a Variable Group for your Pipeline](#create-a-variable-group-for-your-pipeline)
+  - [Variable Descriptions](#variable-descriptions)
+- [Provisioning resources using Azure Pipelines](#provisioning-resources-using-azure-pipelines)
+  - [Create an Azure DevOps Service Connection for the Azure Resource Manager](#create-an-azure-devops-service-connection-for-the-azure-resource-manager)
+  - [Create the IaC Pipeline](#create-the-iac-pipeline)
+- [Create an Azure DevOps Service Connection for the Azure ML Workspace](#create-an-azure-devops-service-connection-for-the-azure-ml-workspace)
+- [Set up Build, Release Trigger, and Release Multi-Stage Pipeline](#set-up-build-release-trigger-and-release-multi-stage-pipeline)
+  - [Set up the Pipeline](#set-up-the-pipeline)
+- [Further Exploration](#further-exploration)
+  - [Deploy the model to Azure Kubernetes Service](#deploy-the-model-to-azure-kubernetes-service)
+  - [Deploy the model to Azure App Service (Azure Web App for containers)](#deploy-the-model-to-azure-app-service-azure-web-app-for-containers)
+  - [Example pipelines using R](#example-pipelines-using-r)
+  - [Observability and Monitoring](#observability-and-monitoring)
+  - [Clean up the example resources](#clean-up-the-example-resources)
+- [Next Steps: Integrating your project](#next-steps-integrating-your-project)
+  - [Additional Variables and Configuration](#additional-variables-and-configuration)
+    - [More variable options](#more-variable-options)
+    - [Local configuration](#local-configuration)
 
-* Fork this repository if there is a desire to contribute back to the repository else
-* Use this [code template](https://github.com/microsoft/MLOpsPython/generate) which copies the entire code base to your own GitHub location with the git commit history restarted. This can be used for learning and following the guide.
+## Setting up Azure DevOps
 
-This repository contains a template and demonstrates how to apply it to a sample ML project ***diabetes_regression*** that creates a linear regression model to predict the diabetes.
+You'll use Azure DevOps for running the multi-stage pipeline with build (CI), model training, and scoring service release (CD) stages. If you don't already have an Azure DevOps organization, create one by following the instructions at [Quickstart: Create an organization or project collection](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops).
```

Comment:
I would leave out the "(CI)" and "(CD)" indicators. CI isn't exactly equivalent to build, and CD is a particular form of release strategy. The difference isn't all that meaningful to someone just trying to get started with MLOps.